### ISSUE TITLE
Feature > Add support additional client attributes

### ISSUE THREAD

**Issue by kjake** [Kane610/aiounifi#95]
I'm not great at python, but it looks like there are many attributes missing from the clients "library" here, which ultimately could be useful to have exposed, especially upstream in Home Assistant, to drive some advanced automations.  I'm very interested in having access to `powersave_enabled`, but could also use `idletime` and `_last_seen_by_uap`.

[Here's](https://pkg.go.dev/golift.io/unifi#Client) an API implementation in Go, which has everything I know of documented.  I couldn't find this officially for some reason.

Could you add these?  Thank you!

**Comment by Kane610** [comment]
Hello! I can have a look but I'm right now busy with other stuff. As the release schedule for home assistant has been adapted to cover Christmas vacations this will not be available prior to the 2022.2 release.

Please clarify your use cases for the different attributes as that will help when adding support for them on hass side.

**Comment by kjake** [comment]
Thanks for the consideration!  If I understood how to develop for Home Assistant better, I definitely would have proposed this through a PR instead of an issue - as that would have allowed me to prove its worth as well, but, while I can overload the unifi integration via `custom_components`, I'm not sure how to deal with overriding `aiounifi`.

Anyway, my very specific use case is that I'd like to turn on/off a light at my desk based on whether I'm at the desk.  I don't have any gadgets to detect vibration of my desk to trigger this, so I started looking at whether the UniFi controller considers my laptop to be 'away' in Home Assistant if it is in Sleep - and it turn out that it does not, but, it does enter Powersave mode, and I observed that `powersave_enabled` changes state for my laptop client based on whether it is in Sleep, or not; therefore, I could have Home Assistant turn on/off the light at my desk based on the `powersave_enabled` state of my laptop client.

Thanks!
